jmix-reports/reports-flowui/src/main/java/io/jmix/reportsflowui/view/reportwizard/OutputFormatTools.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.reportsflowui.view.reportwizard;
+
+import com.google.common.collect.ImmutableMap;
+import io.jmix.core.Messages;
+import io.jmix.reports.entity.ReportOutputType;
+import io.jmix.reports.entity.wizard.TemplateFileType;
+import jakarta.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@Component("report_OutputFormatTools")
+@Scope(BeanDefinition.SCOPE_PROTOTYPE)
+public class OutputFormatTools {
+
+    protected Map<TemplateFileType, Map<String, ReportOutputType>> availableOutputFormats;
+
+    @Autowired
+    protected Messages messages;
+
+    @PostConstruct
```

Comment:
@glebfox Why? This bean like many others here in reporting was just transferred from existing reports-ui module..

Reply:
This class still needs refactoring, so it's not a big deal to make it look like any other

Reply:
Okay, I remember that we decided use constructor injection, but I don't remember that we decided not to use the `@PostConstruct` annotation.

I see that Spring [suggests](https://docs.spring.io/spring-framework/reference/core/beans/factory-nature.html#beans-factory-lifecycle-initializingbean) using PostConstruct in their docs:

> We recommend that you do not use the InitializingBean interface, because it unnecessarily couples the code to Spring. Alternatively, we suggest using the [@PostConstruct](https://docs.spring.io/spring-framework/reference/core/beans/annotation-config/postconstruct-and-predestroy-annotations.html) annotation or specifying a POJO initialization method.

Reply:
ok, seems that we need to update internal recommendations for such cases